rotkehlchen/globaldb/handler.py

```diff
@@ -2008,3 +2010,93 @@ def get_exchange_name_from_assetid(exchange: Location, asset_identifier: str) ->
             ).fetchone()
 
         return None if identifier is None else identifier[0]
+
+    @staticmethod
+    def query_location_asset_mappings(
+            filter_query: 'LocationAssetMappingsFilterQuery',
+    ) -> tuple[list[dict[str, str | Location | None]], int, int]:
+        """Returns a tuple with the mappings, their amount according to the filter_query, and total
+        amount without any filter. Mappings are in the form of a list of dicts with the keys,
+        asset, location, and location_symbol."""
+        with GlobalDBHandler().conn.read_ctx() as cursor:
+            mappings_total = cursor.execute(
+                'SELECT COUNT(*) FROM location_asset_mappings',
+            ).fetchone()[0]
+
+            query, bindings = filter_query.prepare(with_pagination=False)
+            mappings_count = cursor.execute(
+                f'SELECT COUNT(*) FROM location_asset_mappings {query}', bindings,
+            ).fetchone()[0]
+
+            query, bindings = filter_query.prepare()
+            cursor.execute(
+                f'SELECT local_id, location, exchange_symbol FROM location_asset_mappings {query}', bindings,  # noqa: E501
+            )
+            return [{
+                'asset': identifier,
+                'location': None if location is None else str(Location.deserialize_from_db(location)),  # noqa: E501
+                'location_symbol': symbol,
+            } for identifier, location, symbol in cursor], mappings_count, mappings_total
+
+    @staticmethod
+    def add_location_asset_mappings(entries: list[LocationAssetMappingUpdateEntry]) -> None:
+        """Adds the given mapping entries of asset identifiers and their symbols in the given
+        location to the location_asset_mappings table.
+
+        May Raise: InputError if the pair of location and exchange_symbol already exist"""
```

Comment:
nitpick, here and below for consistency with the rest of the code.
```suggestion
        May Raise:
        - InputError if any of the pairs of location and exchange_symbol already exist"""
```